### ISSUE TITLE
Files are not displayed and accessible

### ISSUE THREAD

**Issue by SuhoK** [tdurieux/anonymous_github#306]
After anonymizing my GitHub repository, the anonymous GitHub page only shows folders without displaying the files inside them when I access the page. How can I fix this issue?

**Comment by gottardi** [comment]
Hello, First of all, many thanks for this!
I thought about reporting a bug, but seems related to this issue, might even be the very same problem.

I think that updating the repository via the anonymize/edit page **does** work and **does** make the files accessible.

However, for some reason, the browser **cache** keeps showing an old version. I am guessing it is redirecting to the content by using an URL like "?v=NUMBER", which might **show empty folders** or a **previous version** of a file (even after refresh repository refresh and browser refresh with ctrl+F5).

After opening it using another browser, I noticed it was working fine.

In summary, please verify the cache expiration setting!  Visitors would never tell if it was supposed to look like this...

**Comment by tdurieux** [comment]
Closing as a duplicate of #429 ("The requested file is not found" in UI while download works). Please follow that thread.
